test/e2e/check-coverage.js in CI test fails

**Describe the bug**

`test/e2e/check-coverage.js` fails in CI test.

```
> three@0.121.1 test-e2e-cov /home/runner/work/three.js/three.js
> node test/e2e/check-coverage.js

/home/runner/work/three.js/three.js/test/e2e/check-coverage.js:17
for ( var key in files ) {
                 ^

ReferenceError: files is not defined
    at Object.<anonymous> (/home/runner/work/three.js/three.js/test/e2e/check-coverage.js:17:18)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! three@0.121.1 test-e2e-cov: `node test/e2e/check-coverage.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the three@0.121.1 test-e2e-cov script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

If I'm right, this error has happened since #20512 was merged?

**To Reproduce**

1. See any PR which is open
2. Check CI test results

***Code***

`files` is defined by reading `examples/files.js` and  using eval.

https://github.com/mrdoob/three.js/blob/r121/test/e2e/check-coverage.js#L16

```js
eval( fs.readFileSync( './examples/files.js' ).toString() );
```

#20512 changed the statement for `files` in `examples/files.js` from `var` to `const`. `let/const` variable doesn't seem to be exposed to out of eval scope.

So an easy solution may be

```js
eval( fs.readFileSync( './examples/files.js' )
    .toString().replace( 'const files', 'var files' ) );
```

But not sure if this is a robust approach.

**Expected behavior**

`test/e2e/check-coverage.js` should run without error.

**Platform:**

 - Device: Desktop
 - OS: Windows
 - Browser: 
 - Three.js version: dev

--- a/test/e2e/check-coverage.js
+++ b/test/e2e/check-coverage.js
@@ -13,11 +13,13 @@ const S = fs.readdirSync( './examples/screenshots' )
 
 // files.js
 const F = [];
-eval( fs.readFileSync( './examples/files.js' ).toString() );
-for ( var key in files ) {
+// To expose files variable to out of eval scope, we need var statement, not const.
+eval( fs.readFileSync( './examples/files.js' )
+	.toString().replace( 'const files', 'var files' ) );
+for ( const key in files ) {
 
-	var section = files[ key ];
-	for ( var i = 0, len = section.length; i < len; i ++ ) {
+	const section = files[ key ];
+	for ( let i = 0, len = section.length; i < len; i ++ ) {
 
 		F.push( section[ i ] );
 

